lib/cucumber/rails/world.rb

```diff
@@ -7,10 +7,20 @@
   # Test Unit not found
 end
 
+module Cucumber
+  module Rails
+    class << self
+      def remove_rack_test_helpers
+        ENV['CR_REMOVE_RACK_TEST_HELPERS'] == "true"
```

Comment:
Perhaps you can use https://api.rubyonrails.org/classes/ActiveModel/Type/Boolean.html this coercion thing, since this is a Rails add-on, we have Rails available.

Reply:
I usually dislike these things, but as this is a free gain. I'll fix this up (If we didn't have rails I'd just do a `casecmp == 0` or something